lib/datasource/go/index.ts

```diff
@@ -146,12 +146,21 @@ async function getDatasource(goModule: string): Promise<DataSource | null> {
  *  - Call the respective getReleases in github/gitlab to retrieve the tags
  *  - Filter module tags according to the module path
  */
-export async function getReleases({
-  lookupName,
-}: GetReleasesConfig): Promise<ReleaseResult | null> {
+export async function getReleases(
+  config: GetReleasesConfig
+): Promise<ReleaseResult | null> {
+  const { lookupName } = config;
+
+  let res = null;
+
+  logger.trace(`goproxy.getReleases(${lookupName})`);
+  res = await goproxy.getReleases(config);
```

Comment:
I guess, it should have something similar for `getDigest` too

Reply:
To the another hand, it's somewhat orthogonal to `GOPROXY` (https://golang.org/ref/mod#checksum-database)

Reply:
Don't think we need it?

Reply:
I think we do, just wonder whether it's okay to make it in separate PR or not